src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -225,6 +225,7 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
       onClose={() => {
         handleCloseModal({ label: null, subject: modalTitle! })
       }}
+      data-test="addressVerificationModal"
```

Comment:
I think the correct value here is [data-testid](https://testing-library.com/docs/queries/bytestid/):
```suggestion
      data-testid="addressVerificationModal"
```